src/rgw/rgw_asio_frontend.cc

```diff
@@ -302,6 +303,14 @@ unsigned short parse_port(const char *input, boost::system::error_code& ec)
   return port;
 }
 
+bool parse_nodelay(boost::asio::string_view input, boost::system::error_code& ec)
```

Comment:
I like the signature of this function, but it strikes me as odd that it doesn't actually understand the nodelay token, as that's already been parsed?  it seems like that logic from parse_endpoint could be pushed down here, iow

Reply:
No need for this function anymore, done